subjects/ascii-art-reverse.en.md

```diff
@@ -0,0 +1,42 @@
+## ascii-art-reverse
+
+### Objectives
+
+Ascii-art-reverse consists on reversing the process, convertin the graphic representation into a text.
+
+- You will have to create a `txt` file with a graphic representation of a random phrase;
+- The program will have **one** argument being the name of the file created;
+- The program must print this phrase in **normal text**.
+
+This project will help you learn about :
+
+- Client utilities;
+- The go file system(**fs**) API;
+- Ways to receive data;
+- Ways to output data;
+- Manipulation of strings;
+- Manipulation of structures.
+
+### Instructions
+
+- Your project must be written in **GO**;
+- The code must respect the **good practices**;
```

Comment:
I find "good practices" a bit vague

Reply:
i can include some of the good practices, but i didn't because of the questions for the review.
but i could put:
- comments/documentation
- consistent indentation
- avoid obvious comments
-consistent naming scheme(camelCase, under_scores)
- principles DRY or DIE and KISS
-avoid deep nesting(forest of ifs)

Reply:
Nice, maybe these good practices could be a separate page with general principles (I would also add YAGNI and SoC) and Go related only principles (formatting, naming convention) so we could refer to this page.

Reply:
i put it like that because of the review questions
but i can put some good practices:
- commenting/documentation
-  consistent indentation
-  avoid obvious comments
- consistent naming scheme (camelCase, under_scores)
- obey the principles "DRY" (Don't Repeat Yourself) or "DIE" (Duplication is Evil) and KISS ( keep it simple, stupid )
- avoid deep nesting

Reply:
ok ok sound good to me, cool